CONP_DATS_fields.md

```diff
@@ -1,47 +1,53 @@
+|Country|extraProperties - origin - country|**REQUIRED IF**| a dataset was generated by one institution, **OR** multiple institutions in the same country.|(Principal) country where this dataset was created.|
+Derived From| |The sources this dataset is derived from.|The sources this dataset is derived from.|
+|Derived From|extraProperties - derivedFrom|**REQUIRED IF** | the dataset is a derived dataset, the URL of the dataset it is derived from is **REQUIRED**.  The original dataset must also be included as a submodule in the derived dataset.|The name of the source dataset this one is derived from.|
+|Parent dataset id|extraProperties - parent_dataset_id|**REQUIRE IF**| the dataset is a derived dataset, the parent dataset id (as specified in _conp_dataset/.gitmodules_) is **REQUIRED**.|The identifier (DOI) of the source dataset this one is derived from.|
+|Primary Publications|primaryPublications|**RECOMMENDED**| The primary publication(s) associated with the dataset, usually describing how the dataset was produced.| The primary publication(s) associated with the dataset, usually describing how the dataset was produced.|
+|Title|Primary Publications -> Title||The name of the publication, usually one sentence or short description of the publication.|The name of the publication.|
+|Publication Venue|Primary Publications -> Publication Venue||The name of the publication venue where the document is published (if applicable).|The name of the publication venue where the document is published (if applicable).|
+|Authors|Primary Publications -> Authors||Authors of the publication.|Authors of the publication.|
+|Affiliations|Primary Publications -> Authors -> Affiliations||Author affiliations.|Authors affiliations.|
+|Date|Primary Publications -> Dates||Relevant dates for the publication. If you provide a date, it must come with a description of the date.|Relevant dates for the publication. If you provide a date, it must come with a description of the date (i.e.: first submission, final approval, date of publication, ...).|
+|Identifier|Primary Publications -> Identifier|| Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the publication. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Identifier|Primary Publications -> Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying an entity locally to a system or globally.|
+|Identifier Source|Primary Publications -> Identifier -> Identifier Source|**RECOMMENDED**| Primary identifier for the publicationt. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the publication. Provide a *Document Object Identifier (DOI)* if you have one. |
```

Comment:
@PapillonMcGill I am not sure this one is correct. For example, if the DOI was generated by Zenodo, should this be Zenodo in that field? If that is the case, then this field does not hold the DOI but the creator of the DOI, no?

Reply:
This is for the DOI of a publication that the dataset was used to generate.

Reply:
Right. What I meant was the there are the following fields:

1) Primary Publications -> Identifier
2) Primary Publications -> Identifier -> Indentifier
3) Primary Publications -> Identifier -> Indentifier Source
---
1) just the description of the identifier section of the DATS model (to be removed according to @emmetaobrien's comment [here](https://github.com/CONP-PCNO/conp-documentation/pull/53/files#r518957043). 
2) the actual identifier (DOI of the paper)
3) the organisation that created the DOI for the paper

But currently, 3) is described as where we should enter the DOI, which I don't think this is what it is. @emmetaobrien do you agree?

Reply:
Change coming  with next commit. It should clarify.